addons/missionModules/CfgVehicles.hpp

```diff
@@ -98,4 +98,16 @@ class CfgVehicles {
         class Arguments {};
         class Attributes {};
     };
+
+    class GVAR(zeus_banziCharge): Module_F {
+        author = QUOTE(PREFIX);
+        category = QUOTE(PREFIX);
+        scope = 1;
+        scopeCurator = 2;
+        isTriggerActivated = 0;
+        isGlobal = 2;
```

Comment:
Module runs were local so we get the correct attach thing
Gets deleted after, so should be fine